src/lender/adapters/deployer.sol

```diff
@@ -113,34 +111,31 @@ contract AdapterDeployer {
         AuthLike(clerk).deny(address(this));
     }
 
-    function initMKR(address lenderDeployer_, address mgr_, address mkrSpotter_, address mkrVat_, address mkrJug_, address mkrUrn_, address mkrLiq_, address mkrEnd_, uint matBuffer_) public {
-        require(mkrDeployer == msg.sender);
+    function deployMgr(address lenderDeployer_, address dai_, address daiJoin_, address end_, address vat_, address vow_, address urn_, address liq_) public {
```

Comment:
as you are passing arguments to a public function we need auth here -> mkrDeployer check